fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -358,6 +360,9 @@ public Void visitSlot(SlotRef node, Scope scope) {
             ResolvedField resolvedField = scope.resolveField(node);
             node.setType(resolvedField.getField().getType());
             node.setTblName(resolvedField.getField().getRelationAlias());
+            // help to get nullable info in Analyzer phase
+            // now it is used in creating mv to decide nullable of fields
+            node.setNullable(resolvedField.getField().isNullable());
```

Comment:
is `resolvedField.getField().isNullable()`  always correct?

Reply:
now fields from base table, inner join, outer join are supported.
fields from expression use the default nullable deduction logica.

Reply:
yes